src/System.Management.Automation/engine/CommonCommandParameters.cs

```diff
@@ -232,6 +232,22 @@ public string PipelineVariable
             set { _commandRuntime.PipelineVariable = value; }
```

Comment:
@KirkMunro, your last commit had 4 failures in `PowerShell-CI-linux`
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can enable breakpoints
```powershell
Exception calling "EnableBreakpoint" with "1" argument(s): "The background process reported an error with the following message: Remote transport error: ."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 106
106:                 $bp = $jobRunspace.Debugger.EnableBreakpoint($bp)                

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can remove breakpoints
```powershell
Exception calling "GetBreakpoints" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 120
120:             foreach ($bp in $jobRunspace.Debugger.GetBreakpoints()) {

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Returns an empty collection when there are no breakpoints
```powershell
Exception calling "GetBreakpoints" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 126
126:             $jobRunspace.Debugger.GetBreakpoints() | Should -HaveCount 0

```
Breakpoint SDK Unit Tests.Handling errors while managing breakpoints in a remote runspace via the SDK.Error occurred in Context block
```powershell
Exception calling "SetCommandBreakpoint" with "1" argument(s): "The runspace state is not valid for this operation."
At /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1:153 char:13
```